bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/observability/metrics/MetricRegistryImpl.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
```

Comment:
To preserve the old commits, the old `MetricRegistry.java` was renamed to `MetricRegistryImpl.java` with ba1f8a769e58bd6c370eacc2f90dd45019a7325e.  Here this detected as a new file. However, the IDE is displaying the old commits.

![Screenshot 2021-06-14 at 19 01 55](https://user-images.githubusercontent.com/26219651/121902208-bc90ab80-cd44-11eb-9e69-69949b757cf7.png)